.gitignore

```diff
@@ -7,6 +7,10 @@
 # Ignore bundler config.
 /.bundle
 
+# Ignore floo files
+.floo
+.flooignore
```

Comment:
@csexton Yeah, it is -- it takes a little getting used to. There are some weird bits about it -- like if I'm running guard and Sean saved a file, it would trigger my guard to run. You also have to remember to just have one person do the commit (I think?) and there's some other "not stepping on each others' toes" to keep in mind.

But performance-wise, it was awesome. I was using Atom.io and Sean used Sublime and it seemed to work pretty seamlessly. I guess you can simulcast your webcam with it (might require the paid version tho?). The editing appears similarly to Google Docs -- you see multiple cursors and the text comes in concurrently from all users.

The one thing that Sean mentioned he disliked (I kinda agree with this too) is that, even though you're both editing the same codebase, you can be working in different files, which in a collaborative sense is fine, but from a "pair-programming" perspective it sorta undermines that process. Next time we do this we might just force ourselves to work in the same file, intentionally.